.github/workflows/push-jar-maven.yml

```diff
@@ -0,0 +1,44 @@
+name: Upload Jar to Maven
```

Comment:
What is the reason to have it as a separate workflow and not as part of CD.yml?

Reply:
> What is the reason to have it as a separate workflow and not as part of CD.yml?

It can be but one issue I faced was with the OS that it runs on. gpg signing has some [issues](https://github.com/gaiksaya/security/runs/686751935?check_suite_focus=true) with ubuntu-latest where as it works fine with ubuntu-16.04

Reply:
What is the downside of changing CD to run on the version of ubuntu where gpg works? Can you confirm with github or ubuntu communities why gpg does not work on the latest.